include/z64save.h

```diff
@@ -354,13 +354,13 @@ typedef enum {
 #define EVENTCHKINF_67 0x67
 #define EVENTCHKINF_68 0x68
 #define EVENTCHKINF_69 0x69
-#define EVENTCHKINF_6A 0x6A
+#define EVENTCHKINF_TALON_WOKEN_IN_KAKARIKO 0x6A
 
 // 0x6B
-#define EVENTCHKINF_6B_INDEX 6
-#define EVENTCHKINF_6B_SHIFT 11
-#define EVENTCHKINF_6B_MASK (1 << EVENTCHKINF_6B_SHIFT)
-#define EVENTCHKINF_6B ((EVENTCHKINF_6B_INDEX << 4) | EVENTCHKINF_6B_SHIFT)
+#define EVENTCHKINF_TALON_RETURNED_FROM_KAKARIKO_INDEX 6
```

Comment:
(afaik Talon never goes back to Lon-Lon after you woke him up as adult)

Reply:
He does, if you talk to him after getting epona, he will move to the stable at Lon Lon